underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -117,15 +118,23 @@ public OutputStream create(String path) throws IOException {
     return null;
   }
 
+  // Same as create(path)
+  @Override
+  public OutputStream create(String path, PermissionStatus ps) throws IOException {
+    LOG.debug("Create with permission status is not supported with OSSUnderFileSystem. Permission "
+        + "Status will be ignored.");
+    return create(path);
+  }
+
   @Override
   public OutputStream create(String path, int blockSizeByte) throws IOException {
-    LOG.warn("blocksize is not supported with OSSUnderFileSystem. Block size will be ignored.");
+    LOG.debug("blocksize is not supported with OSSUnderFileSystem. Block size will be ignored.");
```

Comment:
yes, this was intended. See ALLUXIO-2017, ALLUXIO-2018.
Now those are all removed anyways.